root/etc/cont-init.d/50-config

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# create folders
+mkdir -p /config/www/xbackbone
+
+# copy xbackbone files
+if [ ! -e "/config/www/xbackbone/index.php" ]; then
+    cp -R /app/xbackbone/* /config/www/xbackbone
+    rm -rf /app/xbackbone
```

Comment:
This feels a bit pointless as it'll only do it on first run and removing the files doesn't really achieve anything.

Reply:
Thanks. Fixed